src/pybind/mgr/cephadm/services/nvmeof.py

```diff
@@ -43,6 +44,64 @@ def config(self, spec: NvmeofServiceSpec) -> None:  # type: ignore
         # this may raise
         self.mgr._check_pool_exists(spec.pool, spec.service_name())
 
+    def configure_tls(self, spec: NvmeofServiceSpec, daemon_spec: CephadmDaemonDeploySpec) -> None:
+        """
+        Configure TLS and mTLS files for the NVMeoF daemon.
+
+        - Always attaches server_cert/server_key if TLS is enabled.
+        - If mTLS (enable_auth) is enabled, also attaches client_cert, client_key, and root_ca_cert.
+        - Supports both cephadm-signed and user-provided certificates.
+        """
+
+        if not spec.ssl:
+            self.mgr.log.info("TLS for nvmeof gw is disabled.")
```

Comment:
we might want to also include the service name in this error, in case users have multiple nvmeof services

Reply:
Fixed